packages/@okta/vuepress-site/docs/concepts/ie-intro/index.md

```diff
@@ -1,26 +1,22 @@
 ---
-title: Identity engine
+title: Identity Engine
 meta:
   - name: description
-    content: Identity engine offers customizable building blocks that can support dynamic, app-based user journeys. Find out more about Identity engine, why you would use it, and how to upgrade your org.
+    content: Identity Engine offers customizable building blocks that can support dynamic, app-based user journeys. Find out more about Identity Engine, why you would use it, and how to upgrade your org.
 ---
 
-# Identity engine
+# Identity Engine
 
 <ApiLifecycle access="ie" /><br>
 <ApiLifecycle access="Limited GA" />
 
-## About Identity engine
+## About Identity Engine
 
-Identity engine allows organizations to customize their Okta cloud components to satisfy an unlimited number of identity use cases. Instead of a reliance on pre-defined behavior for authentication, authorization, and enrollment, Identity engine offers customizable building blocks that can support dynamic, app-based user journeys.
+The Okta Identity Engine is a platform service that powers flexible and customizable access experiences, which allows enterprises to build access experiences tailored to their organizational needs. Just one of the many goals of Okta Identity Engine is that customers are no longer bound to any one way of identifying, authorizing, enrolling, and issuing access to users. Instead, they can customize and extend each of these steps, which delivers more flexible and customizable access experiences, including passwordless sign-in, improved per app experiences, enhanced context for secure authentication, and more.
 
-Identity engine provides greater fine-grained control when compared to traditional access flows. It enforces custom requirements at every point in the end user's sign-in experience &mdash; from registration and authentication to enrollment and authorization.
+## Why use Identity Engine?
 
-Using the Admin Console, you can direct the user experience based on user type, group membership, and target application. Since Identity engine can execute Okta Hooks or redirect to external services, you can also extend that experience further within the security guardrails of Okta.
-
-## Why use Identity engine?
-
-Identity engine provides:
+Identity Engine provides:
```

Comment:
For some reason, I can't add a comment on the "Progressive profiling" bullet. 2 things:

1) For clarity, I think more info around "designated points" would be helpful to the reader. I.e. are these designated points in the authentication process or flow?

2) The link text does not mirror the page it goes to, i.e., it goes to a page called "Manage Profile Enrollment polices". Not sure what best fix is, fyi.